core/iwasm/common/wasm_runtime_common.c

```diff
@@ -4212,6 +4212,12 @@ static V128FuncPtr invokeNative_V128 = (V128FuncPtr)(uintptr_t)invokeNative;
           || defined(BUILD_TARGET_RISCV64_LP64) */
 #endif /* end of defined(_WIN32) || defined(_WIN32_) */
 
+// ASAN is not designed to work with custom stack unwind or other low-level things.
+// > Ignore a function that does some low-level magic.
+// (e.g. walking through the thread's stack bypassing the frame boundaries)
+#if defined(__GNUC__)
```

Comment:
Could you use `/* .. */` to add comments? We usually do that since it is C source file.

Reply:
And CI reported code guideline check failure, the length of a line should be no larger than 80.

Reply:
Thank you, tried clang-format but it didn't help for some reason. Will fix